### ISSUE TITLE
Too many pop-up windows, keep control window?

### ISSUE THREAD

**Issue by magnuspalmblad** [NinoVrolijk/compareMS2#10]
The number of popup windows is too large, and somewhat confusing. Let's keep it simple! No splash screen (move information to "About" in the main window), keep the control window, but make the controls inactive, still allowing the user to look at the help. No pop-ups with just a single message and an "OK" button are needed. Just open a runtime window with progress bars, where the tree appears after three comparisons have been made.

In summary:

- Remove the splash screen. It fills no useful purpose I think.
- Remove the "Submit status" pop-up window ("Congratulations...")
- Remove the "Analysis status" pop-up window.

Put the necessary information shown on the splash screen in "About", and display the status (whether "job submitted", "compareMS2 running" or "generating distance matrices") in the control window, keeping everything in one window.
